docs/src/adoc/transaction.adoc

```diff
@@ -0,0 +1,52 @@
+= Transactions
+
+*jdbi* provides full support for JDBC transactions.
+
+== Transactions in the Fluent API
+
+*Handle* objects provide two ways to open a transaction -- *inTransaction* allows
+you to return a result, and *useTransaction* has no return value.
+
+Both optionally allow you to specify the transaction isolation level.
+
+[source,java,indent=0]
+-------------------------------------------
+include::TransactionTest.java[tags=simpleTransaction]
+-------------------------------------------
+
+Here, we (probably unnecessarily) guard a simple _SELECT_ statement with a
+transaction.
+
+== Transactions in SqlObject API
+
+Similarly, you may declare transactions with SqlObject annotations:
+
+[source,java,indent=0]
+-------------------------------------------
+include::TransactionTest.java[tags=sqlObjectTransaction]
+-------------------------------------------
```

Comment:
You cannot use `@Transaction` with a `@Sql[Whatever]` annotation. `SqlObjectFactory` (lines 105-109) only pays attention to the first `@SqlMethodAnnotation`-annotated annotation on the method----all other are ignored.

This has been a paper cut for users for a long time--I think it's worth fixing.